timemachine/fe/atom_mapping.py

```diff
@@ -44,7 +45,16 @@
 
 
 def get_cores(
-    mol_a, mol_b, ring_cutoff, chain_cutoff, max_visits, connected_core, max_cores, enforce_core_core, complete_rings
+    mol_a,
+    mol_b,
+    ring_cutoff,
+    chain_cutoff,
+    max_visits,
+    connected_core,
+    max_cores,
+    enforce_core_core,
+    complete_rings,
+    enforce_chiral=True,
```

Comment:
please disable the default=True, I want to be as explicit as possible here.

Reply:
Disabled in https://github.com/proteneer/timemachine/pull/919/commits/2465c43df8a0d87809f47bb42cee740bbdbf97a0